packages/react-router-dom/package.json

```diff
@@ -23,12 +23,14 @@
     "index.js",
     "matchPath.js",
     "withRouter.js",
-    "README.md",
```

Comment:
Why'd you remove the README? It gets added anyways, but it's also good to make it explicit, in case we want to switch tooling to yarn or something else that differs from npm's behavior.